src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Handlers/ContentPartHandler.cs

```diff
@@ -205,8 +220,13 @@ async Task IContentPartHandler.ClonedAsync(CloneContentContext context, ContentP
         public virtual Task ImportedAsync(ImportContentContext context, TPart instance) => Task.CompletedTask;
         public virtual Task UpdatingAsync(UpdateContentContext context, TPart instance) => Task.CompletedTask;
         public virtual Task UpdatedAsync(UpdateContentContext context, TPart instance) => Task.CompletedTask;
+        public virtual Task ValidatingAsync(ValidatePartContentContext context, TPart instance) => Task.CompletedTask;
+        public virtual Task ValidatedAsync(ValidatePartContentContext context, TPart instance) => Task.CompletedTask;
```

Comment:
Why adding another overload for `ValidatingAsync` and `ValidatedAsync` we already have these methods below

Reply:
The argument changes. if I replace the parameter, that would be a breaking change and I want to avoid that. so I thought to make the old stuff obsolete and we'll remove them in future releases